mypyc/exceptions.py

```diff
@@ -86,7 +86,7 @@ def split_blocks_at_errors(blocks: List[BasicBlock],
                                 op=variant,
                                 line=op.line)
                 branch.negated = negated
-                if op.line != NO_TRACEBACK_LINE_NO:
+                if op.line != NO_TRACEBACK_LINE_NO and func:
```

Comment:
May this cause troubles is a mypy bug will generate function with `.fullname` set to `None` (or an empty string)?

Also TBH I would rename this argument to `func_name` and write `... and func_name is not None`.